app/services/cities_services.py

```diff
@@ -0,0 +1,36 @@
+from http import HTTPStatus
+
+from flask import jsonify
+from sqlalchemy.orm import Query, Session
+
+from app.configs.database import db
+from app.exceptions.state_exc import StateNotFoundError
+from app.models.state_model import StateModel
+
+
+def get_cities():
+    session: Session = db.session
+
+    base_query: Query = session.query(StateModel)
+
+    states = base_query.all()
+
+    serializer = [{"state": state.name, "cities": state.cities} for state in states]
```

Comment:
This serializer can be an own function that can be called in the controller function.